tests/test_ops.py

```diff
@@ -0,0 +1,70 @@
+"""
+Unit tests for GPI2 operation and its utility functions.
+"""
+import pytest
+import torch
+import jax
+
+import pennylane as qml
+from pennylane import numpy as np
+import tensorflow as tf
+from ionizer.ops import GPI2
+
+jax.config.update("jax_enable_x64", True)
+
+class TestGPI2:
+    """Tests for the Operations ``GPI2``."""
+
+    interfaces = [
```

Comment:
I know it's a bit more tedious, but I recommend to actually separate the interface tests into individual ones (one per interface) and create the arrays within, rather than importing the interfaces up top and using this list and the `interface_array` method. The main reason is that a user does not in principle need to have all the interfaces installed; if they don't, running the test file will fail at import.

For an example, take a look at the gradient tests in the parametrized qutrit operations: https://github.com/PennyLaneAI/pennylane/blob/master/tests/ops/qutrit/test_qutrit_parametric_ops.py#L419